cpp/src/parquet/symbols.map

```diff
@@ -34,5 +35,6 @@
       # a system with an older libstdc++ which doesn't include the necessary
       # c++11 symbols.
       std::*;
+      *std::__once_call*;
```

Comment:
Out of curiosity, why the leading asterisk?

Reply:
There's this symbol that was causing issues, and it starts with `void std::...`:

`void std::__once_call_impl<std::_Bind_simple<std::_Mem_fn<void (std::__future_base::_State_base::*)(std::function<std::unique_ptr<std::__future_base::_Result_base, std::__future_base::_Result_base::_Deleter> ()>&, bool&)> (std::__future_base::_State_base*, std::reference_wrapper<std::function<std::unique_ptr<std::__future_base::_Result_base, std::__future_base::_Result_base::_Deleter> ()> >, std::reference_wrapper<bool>)> >()`

Reply:
I see, thanks.